pytest/tests/sanity/one_val.py

```diff
@@ -30,7 +31,7 @@
     ], ["total_supply", "4060000000000000000000000000000000"]
 ]
 nightly_genesis_change = [
```

Comment:
We can remove original `genesis_change` and replace `nightly_genesis_change` with it. `nightly_genesis_change` was intended to take place after chunk-only producers release long time ago. Then "Makes large-ish number of block producers..." comment becomes non-actual, as `num_block_producer_seats_per_shard` is no-op.